libcaf_net/caf/net/fwd.hpp

```diff
@@ -27,6 +27,8 @@ namespace net {
 
 class multiplexer;
 class socket_manager;
+template <class Application, class IdType>
```

Comment:
```suggestion
template <class Application, class IdType = unit_t>
```

then don't forget to include `fwd.hpp` in `transport_worker.hpp` (the implementation of the template then does *not* declare the default again).